parser/src/ast/random_values.rs

```diff
@@ -0,0 +1,85 @@
+use super::Identifier;
+
+// RANDOM VALUES
+// ================================================================================================
+
+/// Declaration of random values for an AIR. Random values colud be represented by a named
+/// identifier which is used to identify a fixed size array of length `size` and empty `bindings`
+/// vector or by named identifier which is used to identify a `bindings` [RandBinding] vector and
+/// its `size` field.
+///
+/// # Examples
+///
+/// If random values are declared in form
+///
+/// ```airscript
+/// random_values:
+///     rand: [15]
+/// ```
+///
+/// created [RandomValues] instance will look like
+///
+/// `RandomValues{name: "rand", size: 15, bindings: []}`
```

Comment:
nit: Should there be spaces between curly braces (before and after) for clarity? Not a strong opinion, wdyt?